pkg/operator/ceph/cluster/crash/add.go

```diff
@@ -0,0 +1,188 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package crash
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"github.com/rook/rook/pkg/operator/k8sutil"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+const (
+	controllerName = "ceph-crashcollector-controller"
+	// AppName is the value to the "app" label for the ceph-crash pods
+	AppName = "rook-ceph-crashcollector"
+	// NodeNameLabel is a node name label
+	NodeNameLabel = "node_name"
```

Comment:
Is this a general node label that is already defined elsewhere?

Reply:
It is used in the nodedrain contoller, but it's only used for the deployment's selector in each case. It doesn't have to be the same across packages, just within the package. If we did want to standardize, I'd suggest just going with corev1.LabelHostname